### PR TITLE
Increase wasm stack size

### LINKED ISSUE #4381: Bug: CopyLongStringMultipleFilesTest fails in WebAssembly environment
### Kùzu version

master

### What operating system are you using?

WebAssembly

### What happened?

`CopyLongStringMultipleFilesTest` fails in WebAssembly environment.

See: https://github.com/kuzudb/kuzu/actions/runs/11376509953/job/31649163826

If I modify the copy file to copy the files individually, it passes the test:
```
COPY person FROM "dataset/long-string-multiple-files/vpersonstringaa.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringab.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringac.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringad.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringae.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringaf.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringag.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringah.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringai.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringaj.csv"
COPY person FROM "dataset/long-string-multiple-files/vpersonstringak.csv"
```

Also, if I remove some files from the directory and leave only 3 files, the test passes.

I also tried to print the paths at https://github.com/kuzudb/kuzu/blob/fcee67835c441f943c5a44ac4b230be92aec0bd4/src/common/file_system/local_file_system.cpp#L149-L188
It looks like the paths are correctly resolved.

### PR DESCRIPTION
**Pull request by royi-luo**
# Description

Default stack size is 64KB which we probably easily go over

Fixes #4381

# Contributor agreement

- [ ] I have read and agree to the [Contributor Agreement](https://github.com/kuzudb/kuzu/blob/master/CLA.md).

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -143,6 +143,7 @@ if(BUILD_WASM)
     add_link_options(-pthread)
     add_link_options(-sPTHREAD_POOL_SIZE=8)
     add_compile_options(-s DISABLE_EXCEPTION_CATCHING=0)
+    add_link_options(-sSTACK_SIZE=8388608) # 8MB
```
Comment (royi-luo) [inline CMakeLists.txt:146]: @mewim what do you think is a good number for this?

File: src/include/storage/store/column_chunk_data.h [hunk 2]
```diff
@@ -198,9 +198,9 @@ class ColumnChunkData {
     virtual void setToInMemory();
     // numValues must be at least the number of values the ColumnChunk was first initialized
     // with
-    //reverse data and zero the part exceeding the original size
+    // reverse data and zero the part exceeding the original size
```
Comment (royi-luo) [inline src/include/storage/store/column_chunk_data.h:201]: Not sure why clang-format didn't do this in a previous PR

Comment (benjaminwinger) [reply src/include/storage/store/column_chunk_data.h:201]: It was a third-party contributor, so I guess that means the job didn't run (I guess it also got merged without my suggestion to correct it to `reserve` (it's not reversing any data).

File: CMakeLists.txt (hunk 1 above)
Comment (mewim) [reply CMakeLists.txt:146]: Yeah I think this is fine. The default stack size in most Linux distro is 8MB. But I think `sSTACK_SIZE` does accept `-sSTACK_SIZE=8MB`. I think we should change bytes to MB for better readability and the comment can be removed.

**Review by mewim — APPROVED** [review]
LGTM. Just try to change the unit to MB and see if it works.

File: src/include/storage/store/column_chunk_data.h (hunk 2 above)
Comment (royi-luo) [reply src/include/storage/store/column_chunk_data.h:201]: 😞

**Pull request merged**